packages/lesswrong/lib/collections/moderatorActions/collection.ts

```diff
@@ -4,6 +4,15 @@ import { addUniversalFields, getDefaultResolvers } from '../../collectionUtils'
 import { getDefaultMutations } from '../../vulcan-core/default_mutations';
 import { forumTypeSetting } from '../../instanceSettings';
 
+/**
+ * Creating a moderator action sets a note on the user's profile for moderators
+ * to see, and triggers a review if necessary.
+ *
+ * Pass currentUser to createMutator to set the moderator who created the
+ * action. Do *not* pass currentUser if the currentUser is the user themselves.
+ * Setting currentUser to null (and validate to false) will create the action as
+ * 'Automod'.
+ */
```

Comment:
@b0b3rt or @Raemon, is this an accurate docstring?

Reply:
Given that you're on team retreat, will merge and then you can tell me if it's wrong and I'll revisit. (I'm pretty confident in it.)